micrometer-core/src/main/java/io/micrometer/core/instrument/binder/tomcat/TomcatMetrics.java

```diff
@@ -153,6 +153,24 @@ private void registerThreadPoolMetrics(MeterRegistry registry) {
                     .tags(allTags)
                     .baseUnit(BaseUnits.THREADS)
                     .register(registry);
+
+            Gauge.builder("tomcat.connections.current", mBeanServer,
+                    s -> safeDouble(() -> s.getAttribute(name, "connectionCount")))
+                    .tags(allTags)
+                    .baseUnit(BaseUnits.CONNECTIONS)
+                    .register(registry);
+
+            Gauge.builder("tomcat.connections.keepalive.count", mBeanServer,
```

Comment:
For consistency, I'm thinking `tomcat.connections.keepalive.current` is a better name